v2/api/network/customizations/route_table_extensions.go

```diff
@@ -167,7 +168,20 @@ func fuzzySetRoute(route genruntime.ARMResourceSpec, embeddedRoute reflect.Value
 	if err != nil {
 		return errors.Wrap(err, "unable to check that embedded route is the same as route")
 	}
-	if string(embeddedRouteJSON) != string(routeJSON) {
+
+	var embeddedRouteJSONMap map[string]interface{}
```

Comment:
Rather than duplicate this code in each extension, can we write a common helper?

`func FuzzyEqualityComparison(a any, b any) (bool, error)`, or something?

Reply:
Done